src/cpu/ref_convolution_int8.cpp

```diff
@@ -35,7 +35,7 @@ namespace {
 void dequantize(float &d, dim_t g, dim_t C, dim_t c, const float *wei_scales,
         bool with_groups, int wei_mask, const float *src_scales) {
     // scale_idx_mult = 1 for per_channel scales and 0, otherwise
-    const int wei_scale_idx_mult = wei_mask != 0;
+    const int wei_scale_idx_mult = wei_mask > 0;
```

Comment:
would it make sense to normalize mask with `quant_entry_t::get_mask()` method instead?
This would keep the use of -1 as default value an implementation details, and should be more safe to avoid some checks issues (e.g. comparison to 0 or usage of binary oring/xoring).

Reply:
Discussed offline. Agreed on keeping mask as `INT_MIN` and get_mask() returning it to force developers check the scales state before querying the mask and using it as is. `INT_MIN` will prevent from collapsing into positive output for bit operations.

If this brings more issues in the future rather it solves, Mourad suggested to return 12 lowest bits of current mask from `get_mask()` which would simply return `0` was `INT_MIN` default value and restore the behavior that always existed.